content/700-optimize/300-recordings.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: 'Recordings'
+metaTitle: 'Optimize: Recordings'
+metaDescription: "Learn about using Optimize's recording feature."
+tocDepth: 3
+toc: true
+---
+
+The Recordings feature is designed to help developers debug and isolate different sets of queries into distinct sessions, known as recording. This targeted approach enables precise performance analysis and optimization by preventing the mixing of queries from different applications or test rounds, leading to clearer insights and more effective debugging.
+
+## Managing a recording session
+
+You can manually start and stop recording sessions via the [Optimize dashboard](https://optimize.prisma.io) by clicking the **Start Recording** and **Stop Recording** buttons.
```

Comment:
We can also describe what would happen if no queries are detected in the recording sessions. 

Would that count as a recording?

Reply:
Nurul you can't have _empty_ recordings.

If you try to create a new recording with an existing empty recording, it'll reuse that empty recording in that workspace.

Reply:
Great! Thanks for clarifying! 👍